alpha-build-machinery/make/examples/multiple-binaries/Makefile.test

```diff
@@ -51,3 +51,12 @@ test-rpm:
 	[[ ! -d ./_output/ ]]
 	$(MAKE) clean
 .PHONY: test-rpm
+
+test-codegen:
+	$(MAKE) update-codegen
+	$(MAKE) verify-codegen
+	./_output/tools/bin/yq m -i manifests/operator.openshift.io_myoperatorresources.crd.yaml manifest.test.crd.yaml-merge-patch
+	-$(MAKE) verify-codegen 2>/dev/null
```

Comment:
this is not testing that it fails, it only ignores errors
`! $(MAKE) verify-codegen`